src/controllers/app/userController.js

```diff
@@ -95,3 +90,42 @@ exports.CheckCode = function (req, res) {
     }
   });
 };
+
+exports.completeSignup = function (req, res) {
+  const { operationId, userName, password } = req.body;
+  var { userId, code } = jwt.verify(operationId, process.env.JWT_SECRET_KEY);

```

Comment:
change var to const please please please please!!!!!!!!!!

Reply:
done